pareto/utilities/results.py

```diff
@@ -918,20 +918,6 @@ def generate_report(
             to_unit = variable.get_units()
         else:
             to_unit = None
-        # if variable data is not none and indexed, update headers to display unit
```

Comment:
Note: For reviewers, these lines are removed because they are duplicates of lines 907-915. They result in duplicated units in headers of the results output.